src/datasets/summarizations/category-topk-coverage.summarization.ts

```diff
@@ -0,0 +1,43 @@
+import * as math from 'mathjs';
+import { Summary } from './types';
+import { CategoricalPoint } from '../metas/types';
+import { cacheSummaries, createOrdinalText } from './utils/commons';
+import { formatY } from '../../utils/formatters';
+
+export interface Config {

```

Comment:
should the configs for different summaries have more specific names to differentiate them?

Reply:
That is an intended design. I want the some of the config to be shared between multiple summarization modules. Take `topk` as an example, if you are curious about the the top 3 item summaries, then the coverage of these 3 items can be a follow up to those summaries. This design (shared configs) is widely used in refactor in common configs like "metric" and "metricUnit".